presto-native-execution/presto_cpp/main/types/tests/TestUtils.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
+
+namespace fs = boost::filesystem;
```

Comment:
Why did you give a name to the namespace instead of using "using". Haven't seen much use fo this in Presto/Velox code.

Reply:
Thanks for pointing it out, this was borrowed from the existing code in `ValuesPipeTest.cpp` and there seem to be a few other usages in presto code but `using` would be ideal here. However, since this namespace is only used for the function `current_path()`, removed this namespace declaration here.